debian/control

```diff
@@ -4,19 +4,23 @@ Section: misc
 Priority: optional
 Standards-Version: 3.9.8
 Build-Depends: cmake,
-               debhelper (>= 9),
-               libbrotli-dev,
-               libgif-dev,
-               libgmock-dev,
-               libgoogle-perftools-dev,
-               libgtest-dev,
-               libhwy-dev,
-               libjpeg-dev,
-               libopenexr-dev,
-               libpng-dev,
-               libwebp-dev,
-               pkg-config,
-Homepage: https://gitlab.com/wg1/jpeg-xl
+ debhelper (>= 9),
+ libbrotli-dev,
+ libgif-dev,
+ libgmock-dev,
+ libgoogle-perftools-dev,
+ libgtest-dev,
+ libhwy-dev,
+ libjpeg-dev,
+ libopenexr-dev,
+ libpng-dev,
+ libwebp-dev,
+ pkg-config,
+ doxygen,
+ asciidoc,
+ xmlto,
```

Comment:
What are `xmlto` and `xdg-utils` for?

Reply:
xmlto provides some files required by lintian

xdg-utils provides xdg-mime, which is referenced in plugins/gdk-pixbuf/CMakeLists.txt

Reply:
`JPEGXL_ENABLE_PLUGINS` defaults to false and we don't enable it in the debian/rules, so the plugins/gdk-pixbuf shouldn't even be parsed. Eventually it would be good to build the pluings to a new .deb package but that's a bit out of scope here and would likely require more work.

Reply:
I'll update the commit to remove xdg-utils.